std/fs/ensure_symlink.ts

```diff
@@ -30,6 +27,11 @@ export async function ensureSymlink(src: string, dest: string): Promise<void> {
 
   await ensureDir(path.dirname(dest));
 
+  const srcStatInfo: Deno.FileInfo | undefined = await Deno.lstat(src);
+  const srcFilePathType: PathType | undefined = getFileInfoType(
+    srcStatInfo,
+  );
```

Comment:
@behnammodi what's the reason for adding explicit types here? It's more code and I'm sure IDEs infer it without a problem. Please revert.